### ISSUE TITLE
Problem/error on first run.

### ISSUE THREAD

**Issue by jakerules** [overviewer/Minecraft-Overviewer#1196]
![image](https://cloud.githubusercontent.com/assets/6377830/5696731/2e848874-99a8-11e4-97c5-3de82121f913.png)

[15:18:43 INFO]: Current time is Jan 11, 2015 3:18:43 PM
[15:18:43 INFO]: System.getProperty('os.name') == 'Windows 7'
[15:18:43 INFO]: System.getProperty('os.version') == '6.1'
[15:18:43 INFO]: System.getProperty('os.arch') == 'amd64'
[15:18:43 INFO]: System.getProperty('java.version') == '1.7.0_71'
[15:18:43 INFO]: System.getProperty('java.vendor') == 'Oracle Corporation'
[15:18:43 INFO]: System.getProperty('sun.arch.data.model') == '64'

Help.....

**Comment by ion1** [comment]
Do any of the directory names in .minecraft/saves have non-ASCII characters, potentially §?

**Comment by CounterPillow** [comment]
I took a look into this: the directory names aren't the issue, the level names are. Minecraft doesn't let you enter `§` as a character in the level name but other non-ASCII characters such as `ä` work fine.

So I had a play with this using gzip and okteta (KDE hex editor).

```
cat level.dat | gzip -d > ayylmao
```

Changed one of the bytes in the level name to `A7`, then saved it again using

```
gzip -c ayylmao > level.dat
```

However, the output I got from Overviewer is slightly different. Notably, it crashes in the NBT loader rather than the save listing function.

```
Traceback (most recent call last):
  File "overviewer.py", line 603, in <module>
    ret = main()
  File "overviewer.py", line 227, in main
    list_worlds()
  File "overviewer.py", line 572, in list_worlds
    worlds = world.get_worlds()
  File "/home/fratti/Projekte/mco_fork/Minecraft-Overviewer/overviewer_core/world.py", line 785, in get_worlds
    info = nbt.load(world_dat)[1]
  File "/home/fratti/Projekte/mco_fork/Minecraft-Overviewer/overviewer_core/nbt.py", line 29, in wrapper
    return func(fileobj, *args)
  File "/home/fratti/Projekte/mco_fork/Minecraft-Overviewer/overviewer_core/nbt.py", line 37, in load
    return NBTFileReader(fileobj).read_all()
  File "/home/fratti/Projekte/mco_fork/Minecraft-Overviewer/overviewer_core/nbt.py", line 193, in read_all
    raise CorruptNBTError("could not parse nbt: %s" % (str(e),))
CorruptNBTError: could not parse nbt: 'utf8' codec can't decode byte 0xa7 in position 5: invalid start byte
```

I probably botched up the UTF-8 encoding some way. However, what I learned along the way is that 1) This is probably a bug introduced by using mods or other trickery for fancy coloured world names, 2) the entire save listing thing is terribly fragile for what it does and can easily be replaced by `ls ~/.minecraft/saves/` (or `dir %APPDATA%\.minecraft\saves`) minus the level names. And the level names are what's causing the issue, and we don't really use them anywhere else.

So opinions time from the other Overviewer contributors and Overviewer users: should we just can the world listing "feature" and be done with it?

**Comment by eminence** [comment]
Closed by #1477
